services/gateway/tests/constants/registerApi.js

```diff
@@ -0,0 +1,381 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const expectedResponseForRegisterApi = {
```

Comment:
Ideally, this should be constructed dynamically based on the configuration in `PATH_API_MAPPINGS` in `services/gateway/routes.js` and `getSocketNamespaces` in `services/gateway/namespaces.js`. Please add a TODO for the future. Should be fine as is for now.